src/components/ErrorBoundary.test.tsx

```diff
@@ -28,26 +28,26 @@ afterEach(() => {
   jest.clearAllMocks()
 })
 
-test('renders auth session error and navigate to logout after 5 seconds', () => {
-  const navigateMock = jest.fn().mockImplementation(() => ({}))
-  useNavigateMock.mockReturnValue(navigateMock)
-  isRouteErrorResponseMock.mockReturnValue(true)
-  routeErrorMock.mockReturnValue({
-    status: 401,
-    statusText: 'Unauthorized',
-    data: 'Unauthorized',
-  })
-
-  render(<ErrorBoundary />)
-
-  expect(
-    screen.getByText(/Your session has expired. Please login again./i)
-  ).toBeInTheDocument()
-
-  jest.advanceTimersByTime(5000)
-
-  expect(navigateMock).toHaveBeenCalledWith(Routes.AUTH_LOGOUT)
-})
+// test('renders auth session error and navigate to logout after 5 seconds', () => {
```

Comment:
Id rather delete blocks than comment them out. IF you need them again you can find it in git history